lib/backpex/live_resource.ex

```diff
@@ -121,6 +121,11 @@ defmodule Backpex.LiveResource do
   @callback return_to(socket :: Phoenix.LiveView.Socket.t(), assigns :: map(), action :: atom(), item :: map()) ::
               binary()
 
+  @doc """
+  Customizes the label of the button for creating a new item. Defaults to "New %{resource}".
+  """
+  @callback create_button_label() :: binary()
```

Comment:
Should we pass certain parameters to the callback function, e.g. the `assigns`?

Reply:
Let's add it when we need it? (YAGNI)